omnibus/package-scripts/updater-deb/postinst

```diff
@@ -98,15 +100,17 @@ set -e
 
 # Set proper rights to the dd-agent user
 chown -R dd-agent:dd-agent ${CONFIG_DIR}
+chmod -R g+rw ${CONFIG_DIR}
 chown -R dd-agent:dd-agent ${LOG_DIR}
-chown -R dd-agent:dd-agent ${INSTALL_DIR}
-chown -R dd-agent:dd-agent ${PACKAGES_DIR}
-chown -R dd-agent:dd-agent ${PACKAGES_LOCK_DIR}
+chmod -R g+rw ${LOG_DIR}
+chown -R dd-updater:dd-updater ${INSTALL_DIR}
+chown -R dd-updater:dd-updater ${PACKAGES_DIR}
+chown -R dd-updater:dd-updater ${PACKAGES_LOCK_DIR}
 
 chmod -R 755 ${PACKAGES_DIR}
 # Lock_dir is world writable as any application with a tracer injected
 # needs to write the PID
-chown -R 666 ${PACKAGES_LOCK_DIR}
+chmod -R 766 ${PACKAGES_LOCK_DIR}
```

Comment:
After a bit of testing, I think this is still not permissive enough and should be 777 actually otherwise injected application won't be able to create lock directories

Reply:
I had in mind that the updater would own each directories of its managed package: ex if it installs apm-java, it creates the lock directory, watch it and delete

Reply:
Ah, this is not the case currently though

Reply:
yes, well the locking part and interaction with injector is not implemented/functional yet

Reply:
Thanks for checking it out, turns out w+x is required to write files in the directory too, not just subdirectories